src/main/java/billgates/view/gui/CustomTitleBorder.java

```diff
@@ -5,12 +5,13 @@
 import javax.swing.border.LineBorder;
 import javax.swing.border.TitledBorder;
 import java.awt.*;
+import java.util.ArrayList;
+import java.util.List;
 
-public class CustomTitleBorder extends TitledBorder {
-
+public class CustomTitleBorder extends TitledBorder{
     public static final int DEFAULT_BORDER_THICKNESS = 3;
     public static final int DEFAULT_EMPTY_BORDER_THICKNESS = 7;
-    public static final Color DEFAULT_BORDER_TEXT_COLOR = new Color(220, 120, 150);
+    public static final Color borderTextColor = new Color(220, 120, 150);
```

Comment:
I don't know what's going on here, but why did you change `DEFAULT_BORDER_TEXT_COLOR` to be `borderTextColor`?

Reply:
Fix it!